pylibdmtx/pylibdmtx.py

```diff
@@ -1,17 +1,19 @@
 from __future__ import print_function
 
+import ctypes
 from collections import namedtuple
 from contextlib import contextmanager
-from ctypes import byref, cast, string_at
+from ctypes import byref, cast, string_at, c_char_p
 
 from .pylibdmtx_error import PyLibDMTXError
 from .wrapper import (
     c_ubyte_p, dmtxImageCreate, dmtxImageDestroy, dmtxDecodeCreate,
     dmtxDecodeDestroy, dmtxRegionDestroy, dmtxMessageDestroy, dmtxTimeAdd,
     dmtxTimeNow, dmtxDecodeMatrixRegion, dmtxRegionFindNext,
     dmtxMatrix3VMultiplyBy, dmtxDecodeSetProp, DmtxPackOrder, DmtxProperty,
-    DmtxUndefined, DmtxVector2, EXTERNAL_DEPENDENCIES
-)
+    DmtxUndefined, DmtxVector2, EXTERNAL_DEPENDENCIES,
+    DmtxSymbolSize, DmtxScheme, dmtxEncodeSetProp, dmtxEncodeDataMatrix, dmtxImageGetProp, dmtxEncodeCreate,
+    dmtxEncodeDestroy)
```

Comment:
I think that both this module and the `create_datamatrix` script can be improved by having lists of options for both sizes and encoding scheme.

I suggest the addition of the following in this file:

```
ENCODING_SCHEME_PREFIX = 'DmtxScheme'
ENCODING_SIZE_PREFIX = 'DmtxSymbol'

ENCODING_SCHEME_NAMES = sorted(
    n.name[len(ENCODING_SCHEME_PREFIX):] for n in DmtxScheme
)
ENCODING_SIZE_NAMES = sorted(
    n.name[len(ENCODING_SIZE_PREFIX):] for n in DmtxSymbolSize
)
```